### DISCUSSION TITLE
Live Share and LaTeX Workshop

### DISCUSSION THREAD

**Discussion by PierreMarchand20** [James-Yu/LaTeX-Workshop#3217] [Q&A]
I saw that VSCode Live Share is not supported according to the FAQ, but I am wondering why. Will it be possible in the future ? This could be very practical for working on the same document. Especially the pdf viewer and the build command.

**Comment by jlelong** [comment]
Having LaTeX-Workshop work with Live Share would be really great. See #2228 for a long discussion on the difficulties to support Live Share. One major difficulty is that Live Share does not support sharing binary files, see MicrosoftDocs/live-share/issues/3372. They plan to support binary files but it is not part of the last release which goes back to June 2020.

**Reply by PierreMarchand20** [reply]
Actually it seems to work in the current version: in one of my live share session 
![image](https://user-images.githubusercontent.com/15056323/160558798-ed1b5f5f-87da-4438-8f86-77caa16ffe54.png)

**Comment by krokosik** [comment]
@PierreMarchand20 see the new version with added Live Share support

**Comment by GeraultTr** [comment]
Thanks for the hard work on 1290cdb7ddc9d25c0f4e54ba65574f3e62d8c62e
However, even though I allowed the port sharing and approved the workspace, pdf doesn't seem to load (pdf tab opens but stays empty). Server launching live session works perfectly aside from this.
Which step did I miss in the wiki from host side?
